src/v2/components/UI/GenericButton/index.js

```diff
@@ -55,14 +65,30 @@ export const buttonBorderWidth = x =>
     SMALL: BUTTON_SMALL_BORDER_WIDTH,
   }[buttonSize(x)])
 
+// To-do switch function based on theme
 export const activeMixin = css`
-  border: ${buttonBorderWidth} solid ${translucentGray('bold')};
-  color: ${translucentGray('bold')};
+  border: ${buttonBorderWidth} solid
+    ${props =>
+      props.theme.name === 'default'
+        ? translucentGray('bold')
+        : translucentWhite('bold')};
+  color: ${props =>
+    props.theme.name === 'default'
+      ? translucentGray('bold')
+      : translucentWhite('bold')};
 `
 
+// To-do switch function based on theme
```

Comment:
If you want `TODO`s to show up in ur VSCode use the format `TODO: Foo`

![](http://static.damonzucconi.com/_capture/cInzYtlsmh3y.png)